### ISSUE TITLE
微软的社会责任报告中可以增加一些关于996的内容了～

### ISSUE THREAD

**Issue by firedom** [MSWorkers/support.996.ICU#374]
Apple、Intel与Microsoft等企业会在自己的供应商责报告中声明公司为改善世界环境做的努力以及自己应负的责任[1][2][3]，比如拒绝使用冲突矿产、打击非自愿劳工现象、支持996运动[4]等。这在一定程度上可以认为是美国的一些企业比中国的一些企业更加有社会责任感，确实在努力改善人类生存环境。反观国内的一些企业，非但不承认自己的管理有问题，反而还狡辩员工不主动加班是对自己的不负责[5]。
此时如果有其他有社会责任感的实体加入进来，就必须有一个制度约束大家的行为，Anti996许可证就是一个能满足这种需求的工具，其他公司可以在此基础上完善相应的制度，并在其力所能及的范围内为改善世界环境作出贡献。

[1] https://www.apple.com/supplier-responsibility/
People come first.In everything we do.

[2]https://www.intel.cn/content/www/cn/zh/corporate-responsibility/supply-chain.html
Building a Responsible, Resilient, and Reliable Supply Chain

[3] https://www.microsoft.com/en-us/corporate-responsibility/
“We are grounded in creating local economic opportunity in every community, helping to unlock the power of technology to address our customers’ most pressing challenges.”

[4] https://github.com/MSWorkers/support.996.ICU
Microsoft and GitHub Workers Support 996.ICU

[5] https://tech.sina.com.cn/2019-04-12/doc-ihvhiewr5343483.shtml
马云：加入阿里，你要做好准备一天12个小时，否则你来阿里干什么？

https://github.com/kattgu7/Anti-996-License/issues/46#issuecomment-486270220

**Comment by firedom** [comment]
工作996，生病ICU
工作007，生病RIP

**Comment by Tedko** [comment]
如果微软官方能有任何支持，那当然非常好

**Comment by organizejs** [comment]
I think we should be careful about giving american companies more credit than Chinese companies. Ultimately, i think of them fairly similarly, they are both about maximizing profits. Here are a few thoughts:
- American companies do not implement 996, but they get their workers to work long hours using ideology: software engineers the US are often taught to think of themselves as entrepreneurs, or "owners" of a feature or product. They are often led to act as if they were in constant 'startup-mode' and are encouraged to model themselves off people like Steve Jobs. To be "obsessive" or in the case of Microsoft, to be "customer obsessed". So some people at Microsoft do end up working long hours, because they really buy into it... Managers don't typically care if they work from home, or if they come into the office at noon. As long as people are working.
- American tech companies (at least the big ones) have a huge workforce of contract workers who suffer from comparable labor conditions. Maybe not quite 996, but they are paid significantly less than full-time employees (FTEs). Because they are paid so much less (sometimes as little as 1/5 or 1/6 the salary of a FTE), these companies can simply hire many of them at low costs to avoid breaking labor laws. 
- American tech companies thrive on hypocrisy. Amazon is a really good example, especially with the most recent call for climate change... Amazon has gotten a lot of favorable press for leading the work in climate change, but if you look at their actual business, it is drenched in hypocrisy. They are constantly courting oil and gas companies to use AWS and even support members of Congress who vote against legislation to curb carbon emissions.

The difference between chinese and american tech companies is that these american multi-national companies really understand public relations. Its a core part of their business to making sure that they are palatable to as wide an audience as possible. But lets not be mistaken, they're goals are the same.

**Comment by Tedko** [comment]
@organizejs  Agree with you

**Comment by waynebaby** [comment]
like a summer bug talking about ice

**Comment by firedom** [comment]
@organizejs I agree you, too. Because I want to investigate about this differences between Chinese and American as much as possible. So could you provide some information or evidence to prove your verdict? I want to know how to operate about some American's entity.

**Comment by firedom** [comment]
> like a summer bug talking about ice

井蛙不可以语于海者，拘于虚也；
夏虫不可以语于冰者，笃于时也；
曲士不可以语于道者，束于教也。
A frog in a well cannot be talked with about the sea;‐‐ he is confined to the limits of his hole.
An insect of the summer cannot be talked with about ice;‐‐ it knows nothing beyond its own season. 
A scholar of limited views cannot be talked with about the Tâo;‐‐ he is bound by the teaching.

https://so.gushiwen.org/guwen/bookv_3270.aspx
https://www.centertao.org/media/The-Writings-of-Chuang-Tzu.pdf

**Comment by organizejs** [comment]
@waynebaby I'm not saying anything about working at a Chinese company - i wouldn't know because i haven't worked at one. I'm just saying what i know about American companies.

**Comment by ff4415** [comment]
> @waynebaby I'm not saying anything about working at a Chinese company - I wouldn't know because I haven't worked at one. I'm just saying what I know about American companies.

----------------

Sound's sad though, but we still need to push progress on it. There is no much experience in China to enforce labors' right. It's the difference with the US. No one knows what will happen before a step. it's a brand new sight. 
There is still room to operate by the Chinese gov and somehow it's glad by the government to see it. Unlike in the US, Chinese laws have lots of flaws.  Each campaign can force the government fixing the laws and I think Chinese gov has willing to do it.
